docs/prompting/zero_shot/s2a.md

```diff
@@ -42,15 +45,17 @@ rewritten_prompt = client.chat.completions.create(
                 Mary then adds 10 more pieces of candy to her collection.
                 Max has 1000 more books than Mary.
                 If Megan has 5 pieces of candy, how many does Mary have in total?
-                """
+                """,
         }
-    ]
+    ],
 )
 
 print(rewritten_prompt.relevant_context)
-# >Mary has 3 times as much candy as Megan. Mary then adds 10 more pieces of candy to her collection.
+"""
+Mary has 3 times as much candy as Megan. Mary then adds 10 more pieces of candy to her collection. Max has 1000 more books than Mary.
```

Comment:
> Max has 1000 more books than Mary.
sometimes the example doesn't work - should we find more deterministic examples?

Reply:
fixed